Corona-Warn-App/src/main/res/values/strings.xml

```diff
@@ -275,13 +275,13 @@
     <string name="settings_notifications_headline_active">Möchten Sie Mitteilungen zu Ihrem Covid-19-Risikostatus aktivieren?</string>
     <string name="settings_notifications_headline_inactive">Mitteilungen sind deaktiviert</string>
     <string name="settings_notifications_body_description">Erlauben Sie automatische Mitteilungen zu Covid-19-Risikostatus.</string>
-    <string name="settings_notifications_body_active">Legen Sie fest, zu welchen Themen Sie informiert beiben möchten.</string>
+    <string name="settings_notifications_body_active">Legen Sie fest, zu welchen Themen Sie informiert bleiben möchten.</string>
```

Comment:
Fixed the typo from "beiben" to "bleiben". However, I suggest to go for "informiert werden möchten", which likely better expresses the intended meaning.